docs/maintain-guides-how-to-chill.md

```diff
@@ -0,0 +1,32 @@
+---
+id: maintain-guides-how-to-chill
+title: Chilling
+sidebar_label: Chilling
+---
+
+Stakers can assume any one of the three states: validating, nominating, or chilling. When a staker
```

Comment:
I think this should also be included on the [Staking](https://wiki.polkadot.network/docs/en/learn-staking) page to avoid confusion.

Reply:
perhaps in the [chilling section](https://wiki.polkadot.network/docs/en/learn-staking#chilling) of the staking page, we could link this new page for more reading :)